cocos/platform/android/CCFileUtils-android.cpp

```diff
@@ -249,6 +268,12 @@ FileUtils::Status FileUtilsAndroid::getContents(const std::string& filename, Res
     } else {
         relativePath = fullPath;
     }
+    
+    if (obbfile)
+    {
+        if (obbfile->getFileData(relativePath, buffer))
+            return FileUtils::Status::OK;
```

Comment:
As the return value type is `bool`, and `FileUtils::Status::OK` is an enum value. So it return incorrect type here.